monitoring/README.md

```diff
@@ -0,0 +1,98 @@
+# Monitoring
+
+This directory contains the monitoring dashboards for Accelerators and vLLM performance metrics. This includes:
+
+- [NVIDIA](grafana/nvidia/nvidia-vllm-dashboard.yaml) GPU metrics with vLLM Performance Metrics.
+- [inputs.env](inputs.env): holds the parameters that need to be updated before deploying the Grafana dashboard.
+
+The Grafana dashboard object is defined as a JSON string as this example:
+```yaml
+...
+spec:
+  instanceSelector:
+    matchLabels:
+      dashboards: grafana
+  folder: "vLLM / GPU Metrics"
+  json: |
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": {
+              "type": "$(DATASOURCE)",
+              "uid": "grafana"
+            },
+            "enable": true,
+            "hide": true
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "type": "dashboard"
+          }
+        ]
+      }
+...
+#remaining of the JSON definition
+```
+
+With this approach, it makes difficult to use kustomize to correctly override the needed parameters so the Dashboard
+can correctly show the metrics deployed in your current namespace. There are two parameters that needs to be updated:
+
+- **NAMESPACE**: the targe namespace where the target model will be deployed.
+- **MODEL_NAME**: the same as defined in your InferenceService, note, it is a regex, so just the starting string of your model name would be enough.
+
+To correctly override it, we will use a helper called `envsubst`, it is available through the `gettext` GNU package.
```

Comment:
Should we add an install command?

Reply:
true, I just think the requirement is a bit hidden without an install command to draw attention to it - what do we usually do for non python requirements?

Reply:
It depends. Usually, we add some reference to where that command (in this case, GNU) can be found, and the user installs it based on their system.